datasets/us/ice_wanted/crawler.py

```diff
@@ -0,0 +1,153 @@
+from zavod import Context
+from xml.etree import ElementTree
+from normality import collapse_spaces
+
+
+def get_element_text(doc: ElementTree, xpath_value: str, to_remove=[]) -> str:
+    """Extract text from each child nodes of an xpath and joins them together
+
+    Args:
+        doc (ElementTree): HTML Tree
+        xpath_value (str):  xpath to extract text from
+        to_remove (list, optional): string to remove in the extracted text.
+    """
+    element_tags = doc.xpath(xpath_value)
+
+    tag_list = []
+    for tag in element_tags:
+        tag_list.append(tag.text_content())
+
+    element_text = "".join(tag_list)
+
+    for string in to_remove:
+        element_text = element_text.replace(string, "")
+
+    return collapse_spaces(element_text.strip())
+
+
+def generate_mw_notes(**kwargs):
+    note_string = f"""Additional Information:
+- Wanted For: {kwargs.get('crime_summary')}
+- Wanted Heading: {kwargs.get('crime_details')}
+- Wanted Category: {kwargs.get("category")}
+- Wanted Status: {kwargs.get("status")}
+- Age: {kwargs.get("age")}
+- Height: {kwargs.get("height")}
+- Weight: {kwargs.get("weight")}
+- Skin tone: {kwargs.get("skin_tone")}
+- Eyes: {kwargs.get("eyes")}
+- Hair: {kwargs.get("hair")}
+- Last Known Location: {kwargs.get('last_known_location')}"""
+
+    return note_string
+
+
+def crawl_person(context: Context, url: str, crime_summary: str):
+    doc = context.fetch_html(url, cache_days=1)
+
+    names = get_element_text(
+        doc,
+        '//div[contains(@class, "field--name-field-most-wanted-name")]//div[contains(@class, "field__item")]',
+    )
+    names_split = names.split(",")
+
+    name = names_split[0]
+    alias = names_split[1:]
```

Comment:
as far as I can tell, the name is actually `surname(s), forename(s)` rather tan `name, aliases`. Or do you have different examples?

There's an alias field which I think we're missing right now.

Reply:
You're right.
Some of them had two names before the comma, eg https://www.ice.gov/most-wanted/garcia-jerez-anibal,  so I though it was an alias